services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersValidationSingleSwFlowSpec.groovy

```diff
@@ -0,0 +1,411 @@
+package org.openkilda.functionaltests.spec.switches
+
+import static org.junit.Assume.assumeTrue
+import static org.openkilda.testing.Constants.WAIT_OFFSET
+
+import org.openkilda.functionaltests.BaseSpecification
+import org.openkilda.functionaltests.helpers.SwitchHelper
+import org.openkilda.functionaltests.helpers.Wrappers
+import org.openkilda.messaging.command.switches.DeleteRulesAction
+import org.openkilda.messaging.error.MessageError
+import org.openkilda.testing.model.topology.TopologyDefinition.Switch
+
+import groovy.transform.Memoized
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.client.HttpClientErrorException
+import spock.lang.Narrative
+import spock.lang.Unroll
+
+@Narrative("""This test suite check the switch validate feature on a single flow switch.
+Description of fields:
+- missing - those meters/rules, which are NOT present on a switch, but are present in db
+- misconfigured - those meters which have different value (on a switch and in db) for the same parameter
+- excess - those meters/rules, which are present on a switch, but are NOT present in db
+- proper - meters/rules values are the same on a switch and in db
+""")
+class MetersValidationSingleSwFlowSpec extends BaseSpecification {
+    @Autowired
+    SwitchHelper switchHelper
+
+    @Unroll
+    def "Switch validation is able to store correct information on a #switchType switch in the proper section"() {
```

Comment:
Can you please put 'proper' in quotes, so that it refers to  the concrete name of the section and not just 'some proper section'

Reply:
Fixed

Reply:
Fixed